### DISCUSSION TITLE
How to add word with special characters in custom dictionary

### DISCUSSION THREAD

**Discussion by fharper** [facelessuser/pyspelling#147] [Q&A]
How to add word with special characters in custom dictionary? Let's say, a word with a `.` like `Node.js` is taken in consideration even if it mind custom dictionary. It will be treated like two words anyway `Node` & `js` . How can I make the spellchecker or Aspell treat it like `Node.js` instead?

**Comment by facelessuser** [comment]
I'll link you to this on the Aspell doc site: http://aspell.net/0.60.7/man-html/Words-With-Symbols-in-Them.html.

With that said, a plugin to filter out such cases could probably work. Depending on how you want to handle it, you may be able to get along using the existing plugin maybe.

Aspell, just won't treat a word with a dot as a whole word, so you can create a filter rule for things like `.js`.
